src/main/java/thePackmaster/cardmodifiers/energyandechopack/EchoedEtherealMod.java

```diff
@@ -0,0 +1,42 @@
+package thePackmaster.cardmodifiers.energyandechopack;
+
+import basemod.abstracts.AbstractCardModifier;
+import basemod.helpers.CardBorderGlowManager;
+import basemod.helpers.CardModifierManager;
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.helpers.GameDictionary;
+import com.megacrit.cardcrawl.localization.LocalizedStrings;
+import org.apache.commons.lang3.StringUtils;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.cards.marisapack.AmplifyCard;
+
+//Basically Ethereal Modifier but it will go at the bottom and won't NL
```

Comment:
Why have this be different from where the word Ethereal normally goes? Is it because you want to avoid adding new lines to cards? It looks to me like all the Echo cards have short enough descriptions that adding a line for Ethereal at the top would be okay. Though it looks like Tear Apart might reach six lines.

Reply:
this plus i find it better if one effect (echo) adds lines at the same place rather than on each side of the card